api/naming/naming.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/docker/swarmkit/api"
 )
 
+const (
+	// PredefinedLabel identifies internally allocated swarm resources
+	PredefinedLabel = "com.docker.swarm.predefined"
```

Comment:
I need to take this back. 
In case the swarm cluster was created with older docker version, the `internal` label will be present in the old `ingress` network's spec. The docker cluster code need to be able to identify only the new predefined networks.
The predefined term was chosen in line with the predefined ("bridge", "host",...) docker engine networks.